addon/components/o-s-s/toggle-switch.ts

```diff
@@ -18,7 +24,9 @@ export default class OSSToggleSwitchComponent extends Component {
   }
 
   @action
-  switchState(event) {
+  switchState(event: MouseEvent): void {
+    event.stopPropagation();
```

Comment:
Maybe add an args `useCapture?: boolean`.This will add a condition to the event capture